tests/integration_tests/resources/dsl/scripts/workflows/test_deployment_id_parameters.py

```diff
@@ -1,11 +1,28 @@
+import json
 import sys
 
-import json
+from cloudify.decorators import workflow
+from cloudify.manager import get_rest_client
+from cloudify.workflows import ctx
 
 
+@workflow
```

Comment:
this isn't actually a workflow function though. The whole script is run (so the workflow function is `script_runner.tasks.execute_workflow`)